include/functions.h

```diff
@@ -981,10 +981,10 @@ CollisionHeader* BgCheck_GetActorMeshHeader(CollisionContext* colCtx, s32 index)
 // void func_800C3C94(void);
 f32 func_800C3D50(s32 arg0, CollisionContext* colCtx, s32 arg2, CollisionPoly** arg3, s32* arg4, Vec3f* pos, Actor* actor, s32 arg7, f32 arg8, s32 arg9);
 // void func_800C3F40(void);
-// void func_800C3FA0(void);
+f32 func_800C3FA0(CollisionContext* colCtx, s32* param_2, Vec3f* param_3);
```

Comment:
```suggestion
f32 func_800C3FA0(CollisionContext* colCtx, s32* arg1, Vec3f* arg2);
```
An upcoming PR will be changing `params` to `args`. If `params` is already there, then it's fine to leave it as it'll be taken care of, but it's best not to introduce `params` from when it wasn't there before, in case there's overlap with that PR.
This comment applies to the function below as well `func_800C40B4`